source/Octopus.Tentacle.Tests.Integration/Support/TentacleTestCaseSourceAttribute.cs

```diff
@@ -44,7 +44,8 @@ public class TentacleTestCaseSourceAttribute : NUnitAttribute, ITestBuilder, IIm
         private readonly NUnitTestCaseBuilder _builder = new();
         
         public const string Net60ClientNet48Service = nameof(Net60ClientNet48Service);
```

Comment:
This category is preserved as it is reference in TeamCity build configurations. Once this has merged, the build configuration can be updated and a subsequent PR can remove the obsolete category.